yjit/src/utils.rs

```diff
@@ -122,12 +122,20 @@ yjit_print_iseq(const rb_iseq_t *iseq)
 
 #[cfg(target_arch = "aarch64")]
 macro_rules! c_callable {
-    (fn $f:ident $args:tt $(-> $ret:ty)? $body:block) => { extern "C" fn $f $args $(-> $ret)? $body };
+    ($(#[$outer:meta])*
```

Comment:
What's outer:meta ???

Reply:
The doc comments outside of functions like `///` are actually sugar for meta attributes like `#[doc="..."]` and this is how you'd match against them.